.travis.yml

```diff
@@ -19,7 +19,7 @@ addons:
       - rsyslog
   mariadb: "10.0"
 
-sudo: false
+sudo: true
```

Comment:
> does Travis set permissions such that only the root/mysql user can read the error log?

Yep.

I think it's worth temporarily slowing down tests if we can figure out the flaky test problem. I'll also set the timeouts in the MySQL connection much lower, to 1 second, which should mean we are more likely to hit the problem quickly and get actionable data.